qna/General-firewall-validation.md

```diff
@@ -0,0 +1,16 @@
+## Question
+How can I test the connection to the fiskaltrust Services in case the Middleware has connection issues?
```

Comment:
fiskaltrust.Services

Reply:
Changed it to be more obvious about our cloud services - I'd rather not use "fiskaltrust.Services" here, because it can be easily confused with the MW 1.2